components/camel-chatscript/src/test/java/org/apache/camel/component/ChatScriptComponentTest.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.test.junit4.CamelTestSupport;
+import org.junit.Test;
+
+public class ChatScriptComponentTest extends CamelTestSupport {
+
+    @Test
+    public void testChatScript() throws Exception {
+        MockEndpoint mock = getMockEndpoint("mock:result");
+        mock.expectedMinimumMessageCount(1);       
+        Thread.sleep(100);
+        assertMockEndpointsSatisfied();
+    }
+
+    @Override
+    protected RouteBuilder createRouteBuilder() throws Exception {
+        return new RouteBuilder() {
+            public void configure() {
+            	  char nullChar = (char) 0;
+                from("ChatScript://localhost:1024/Harry?chatusername=varun&resetchat=true")
```

Comment:
It looks that the scheme is sometime _ChatScript_ and sometime _chatscript_. I would always use _chatscript_ in lower case as the scheme in annotations, examples, services...

Reply:
I have used ChatScript everywhere. I will change it to lowercase and re-commit

Reply:
changed to "chatscript"